scripts/west_commands/runners/bossac.py

```diff
@@ -46,19 +46,22 @@ def create(cls, cfg, args):
                                   port=args.bossac_port, offset=args.offset)
 
     def do_run(self, command, **kwargs):
-        if platform.system() != 'Linux':
-            msg = 'CAUTION: No flash tool for your host system found!'
+        if platform.system() == 'Windows':
+            msg = 'CAUTION: BOSSAC runner not support on Windows!'
             raise NotImplementedError(msg)
 
-        self.require('stty')
-        self.require(self.bossac)
-        cmd_stty = ['stty', '-F', self.port, 'raw', 'ispeed', '1200',
-                    'ospeed', '1200', 'cs8', '-cstopb', 'ignpar', 'eol', '255',
-                    'eof', '255']
+        if platform.system() == 'Linux':
+            self.require('stty')
+            self.require(self.bossac)
```

Comment:
This line still needs to be done on macOS -- it's checking that bossac is installed and on PATH.

Could you please move it after the NotImplementedError if block?